apps/site/test/site_web/controllers/event_controller_test.exs

```diff
@@ -69,6 +69,7 @@ defmodule SiteWeb.EventControllerTest do
     end
   end
 
+  # setup similar test for mobile menu?
```

Comment:
Yeah, probably for testing the layout view https://github.com/mbta/dotcom/blob/eaac9fdb9b32f0c55f64689ec23de01de283a513/apps/site/test/site_web/views/layout_view_test.exs#L32